src/ui/osx/TogglDesktop/test2/AppDelegate.m

```diff
@@ -100,18 +102,10 @@ - (void)applicationWillFinishLaunching:(NSNotification *)not
 	self.lastKnownUserID = 0;
 	self.showMenuBarTimer = NO;
 
-	if ([self updateCheckEnabled])
-	{
-		[[SUUpdater sharedUpdater] setAutomaticallyDownloadsUpdates:YES];
-
-		NSAssert(ctx, @"ctx is not initialized, cannot continue");
-		char *str = toggl_get_update_channel(ctx);
-		NSAssert(str, @"Could not read update channel value");
-		NSString *channel = [NSString stringWithUTF8String:str];
-		free(str);
-		[Utils setUpdaterChannel:channel];
-	}
 	[NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
+
+	NSOperatingSystemVersion osVersion = [[NSProcessInfo processInfo] operatingSystemVersion];
+	unsupportedOS = (osVersion.majorVersion == 10 && osVersion.minorVersion < 15);
```

Comment:
This logic is appeared twice in our codebase. We should respect DRY. We could use the `unsupportedOS` in `UnsupportedNotice`